common-gradle-enterprise-maven-configuration/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -14,5 +14,5 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.6/apache-maven-3.8.6-bin.zip
```

Comment:
Should this normally be done by the wrapper upgrade plugin? Was this in the upgrade procedure to upgrade this file?

Reply:
It should indeed, I just decided to make it part of my update

Reply:
Great, just double checking :)